src/views/pages/globalping.html

```diff
@@ -1639,6 +1639,7 @@
 			}
 
 			function byteLinear (partOne, partTwo, quotient) {
+				// eslint-disable-next-line no-bitwise
 				let color = (('0x' + partOne) * (1 - quotient) + ('0x' + partTwo) * quotient) | 0;
```

Comment:
What is this meant to do? Bitwise or with `0` doesn't do anything.

Reply:
It floors the number with handling NaN case, however yeah i think you are right and this time it is too much, replaced with regular Math.floor()